WickedEngine/wiBacklog_BindLua.cpp

```diff
@@ -126,6 +127,54 @@ namespace wi::lua::backlog
 		wi::backlog::UnblockLuaExecution();
 		return 0;
 	}
+	int backlog_open(lua_State* L)
+	{
+		wi::backlog::Flush();
+		std::string logfile = wi::helper::GetCurrentPath() + "/log.txt";
+		wi::backlog::GetLogFile(logfile);
+#ifdef PLATFORM_WINDOWS_DESKTOP
+		const std::string op = "start \"\" \"" + logfile + "\"";
```

Comment:
For some reason, opening the log file has stopped working and now fails. I need to investigate. 🤔